Topics/Tech_Stacks/TinyMCE.md

```diff
@@ -0,0 +1,39 @@
+## TinyMCE API with ReactJS
+
+### What is TinyMCE API
+TinyMCE API is an free API that allows you to add an WYSIWYG(what you see is what you get) HTML editor on your website. This API essentially makes you textbox interactive for instance users can enter text, image and more in the textbox. This API is especially helpful if your app is heavy on editing or involves a lot of text changes from the user.
```

Comment:
Could you expand on what you mean by "heavy on editing"? Maybe include an example of what types of apps would have this functionality